lib/graph_utils.py

```diff
@@ -33,6 +33,9 @@ def visit_node(node, depth = 0):
             if node.kind == CursorKind.FUNCTION_DECL:
                 # Stops us from going into the function body because we don't care for headers
                 return
+            if node.kind == CursorKind.STRUCT_DECL:
+                # Stops us from going into the struct body because we don't care for headers
+                return
```

Comment:
what about if the top level node is a union?

```c
union foo {
  struct bar;
  struct baz;
};
```
only `foo` should be reported, not `bar` or `baz`.